userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+      3: cvvResultCode
+  databases:
+    warehouse:
+      type: trino
+      url: jdbc:trino://example.net:8080/hive/sales?user=admin' \
+    http://127.0.0.1:8080/1.0/kb/tenants/uploadPluginConfig/killbill-analytics
+----
+
+This configures the following:
+
+[options="header",cols="1,1,1"]
+|===
+|Property Name   |Description   |Default Value  
+//----------------------
+|refreshDelaySeconds   |Specifies the delay in seconds, before starting to refresh analytics tables after an event is received. For workflows with lots of successive events (e.g. create account, add payment method, create payment), this makes sure that the latest state is available when starting the refresh   |10   
+|lockAttemptRetries   |Specifies the number of retries to get the lock   |100   
+|rescheduleIntervalOnLockSeconds   |Specifies the duration in seconds after which the refresh job should be scheduled in case the lock is not available   |10   
+|enablePartialRefreshes   |Specifies whether partial refreshes are allowed. If true, the plugin refreshes only a subset of the analytics tables based on the received event. If false, all the analytics tables are updated.   |true   
+|enableTemplateVariables   |Specifies whether template variables should be allowed in raw SQL queries while creating reports based on SQL queries. Note that this could be prone to SQL injection and should only be enabled in trusted environments.   |false   
+|blacklist   |Specifies a list of account ids to ignore   |-
+|highCardinalityAccounts   |Specifies a list of account ids with a high cardinality (account with 100k+ subscriptions). This is a performance hint to the plugin and tells the plugin to look up individual subscription rows instead of querying everything based on `account_record_id`. |-
+|ignoredGroups   |Specifies a list of https://github.com/killbill/killbill-analytics-plugin/blob/59c282f97960dc59b944ca33e17353bc19d7c842/src/main/java/org/killbill/billing/plugin/analytics/AnalyticsJobHierarchy.java#L59[groups] that should be ignore for refresh. Possible values are `FIELDS`, `INVOICES`, `INVOICE_AND_PAYMENTS`,`OVERDUE`, `OTHER`, `SUBSCRIPTIONS`, `ALL`. Thus, database tables corresponding to these groups will be ignored while performing a refresh.    |-   
+|pluginPropertyKeys   |Specifies a payment plugin name and plugin properties related to the payment plugin to be stored into the analytics tables. So, if a payment plugin populates these values, these are fetched and stored in the `analytics_payment_auths` table.   |1:processorResponse,
+                                                                                  2:avsResultCode,                                                                                                3:cvvResultCode,                                                                                               4:payment_processor_account_id                                                                                                 5:paymentMethod   
```

Comment:
I would put a period after each sentence in the table.

Reply:
Fixed